pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[project]
+name = "svglib"
+version = "1.5.2"
+description = "A pure-Python library for reading and converting SVG"
+readme = "README.rst"
+authors = [
+    { name = "Dinu Gherman", email = "gherman@darwin.in-berlin.de" },
+]
+keywords = [
+    "SVG",
+    "PDF",
+    "reportlab",
+    "conversion",
+    "graphics",
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: End Users/Desktop",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Natural Language :: English",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Documentation",
+    "Topic :: Utilities",
+    "Topic :: Printing",
+    "Topic :: Multimedia :: Graphics :: Graphics Conversion",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Text Processing :: Markup :: XML",
+]
+license = "LGPL 3"
```

Comment:
This might work better:

license = "LGPL-3.0-or-later"

Reply:
Oh that's a fun error... one sec.